project/ParadoxSupport.scala

```diff
@@ -0,0 +1,88 @@
+package org.scalanative.bindgen.build
+
+import com.lightbend.paradox.markdown._
+import com.lightbend.paradox.sbt.ParadoxPlugin.autoImport.paradoxDirectives
+import org.pegdown.Printer
+import org.pegdown.ast.{DirectiveNode, TextNode, Visitor}
+import scala.collection.JavaConverters._
+
+object ParadoxSupport {
+  val paradoxWithCustomDirectives = Seq(
+    paradoxDirectives ++= Seq(
+      { context: Writer.Context ⇒
+        new BindingDependencyDirective(context.location.tree.label,
+                                       context.properties)
+      }
+    )
+  )
+
+  /* Based on the DependencyDirective from Paradox. */
+  case class BindingDependencyDirective(page: Page,
+                                        variables: Map[String, String])
+      extends LeafBlockDirective("binding") {
+    val ProjectVersion     = variables("project.version")
+    val ScalaBinaryVersion = variables("scala.binary.version")
+
+    def render(node: DirectiveNode,
+               visitor: Visitor,
+               printer: Printer): Unit = {
+      node.contentsNode.getChildren.asScala.headOption match {
+        case Some(text: TextNode) =>
+          renderBindingDependency(text.getText, printer)
+        case _ => node.contentsNode.accept(visitor)
+      }
+    }
+
+    def renderBindingDependency(binding: String, printer: Printer): Unit = {
+      val artifactName = s"lib$binding"
+      val artifactId   = s"${artifactName}_native0.3_${ScalaBinaryVersion}"
```

Comment:
`native0.3` is currently hard-coded. I don't know if it is possible to get it from the sbt-scala-native plugin.